### PR TITLE
pet score, invite command

### PR DESCRIPTION
**Pull request by 178619**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -2054,6 +2075,14 @@ async def support_server(self, message, *args):
             text='(ﾉ◕ヮ◕)ﾉ*:･ﾟ✧'
         ).send()
 
+    async def invite(self, message, *args):
+        await Embed(
+            message.channel,
+            user=message.author,
+            title='Here\'s a invite link',
```
Comment (uwuplant) [inline bot.py:2082]: should be "an invite link"

Comment (uwuplant) [inline bot.py:2082]: should be "an invite link"

File: bot.py [hunk 2]
```diff
@@ -643,6 +645,10 @@ def __init__(self, *args, **kwargs):
                     'support': {
                         'function': self.support_server,
                         'desc': 'Have an question about the bot? Use this command'
+                    },
+                    'invite': {
+                        'function': self.invite,
+                        'desc': 'Do you wanna add the bot to your server? Use this command'
```
Comment (uwuplant) [inline bot.py:651]: "Use this command to invite the bot to your server" sounds better imo

File: bot.py (hunk 1 above)
Comment (178619) [reply bot.py:2082]: my grammar 🤦

**Pull request merged**